core/src/main/java/com/venta/VentaApplication.java

```diff
@@ -0,0 +1,107 @@
+package com.venta;
```

Comment:
I think there is missing one more level of package.
com.venta is common for all our modules. There also should be a module name.
I'd say that container would be a good name. Maybe for the module itself as well.